taichi/transforms/lower_access.cpp

```diff
@@ -153,6 +153,18 @@ class LowerAccess : public IRVisitor {
     }
   }
 
+  // TODO: this seems to be redundant
+  void visit(GlobalTensorElementStmt *stmt) override {
+    if (stmt->origin->is<GlobalPtrStmt>()) {
```

Comment:
nit: early return?

Reply:
I am confused about what this means.

Reply:
https://github.com/taichi-dev/taichi/pull/2543/commits/1123d63e7ce115f058222f0af7e21d39add13d0e